forge-game/src/main/java/forge/game/ability/AbilityUtils.java

```diff
@@ -227,6 +227,9 @@ else if (defined.equals("Remembered") || defined.equals("RememberedCard")) {
             for (final Object o : hostCard.getRemembered()) {
                 if (o instanceof Card) {
                     cards.add((Card) o);
+                } else if (o instanceof Spell) {
```

Comment:
I'm not sure if that doesn't cause problems on some script if it finds more than it might expect...
Why not just RememberForCounter$?